packages/cli/package.json

```diff
@@ -193,7 +193,9 @@
       "@loopback/socketio": "^0.6.5",
       "@loopback/monorepo": "^0.5.5",
       "@loopback/tsdocs": "^4.0.5",
-      "@loopback/example-references-many": "^6.0.5"
+      "@loopback/example-references-many": "^6.0.5",
+      "spdy": "^4.0.2",
+      "@types/spdy": "^3.4.5"
```

Comment:
I don't think we need to add `spdy` related deps for the cli. They are set into the project being created.

Reply:
Here I have added spdy dependency in templateDependencies of the cli and not as a part or cli dependencies, so spdy related dependencies will be added in the project being created and not in cli

Reply:
Also adding spdy dependency to a project only when the config (project.isHttp2) is true.
The default value of config is false and is set to true only when the user confirms to the prompt when asked during app creation. otherwise no changes in the app will be made and a normal loopback app will be generated.